source/tests/pt/common.py

```
@@ -257,10 +285,35 @@
                 [nframes, natoms, 3, 3], dtype=GLOBAL_PT_FLOAT_PRECISION, device=DEVICE
             )
         )
-        updated_coord_out = torch.cat(updated_coord_out) if updated_coord_out else None
+        strain_components_out = (
+            torch.cat(strain_components_out)
+            if strain_components_out
+            else torch.zeros(
+                [nframes, 6], dtype=GLOBAL_PT_FLOAT_PRECISION, device=DEVICE
+            )
+        )
+        atom_component_out = (
```

Ruff (0.8.2)

Local variable `atom_component_out` is assigned to but never used

Remove assignment to unused variable `atom_component_out`

(F841)